packages/celotool/src/lib/attestation-service.ts

```diff
@@ -48,6 +58,12 @@ function helmParameters() {
     // TODO(nambrot): Hardcode for now, couldn't figure out how to make it work dynamically
     // DB is exposed as ClusterIP service only
     `--set global.postgresql.postgresqlPassword=password`,
+    `--set geth.genesisFile=${Buffer.from(generateGenesisFromEnv()).toString('base64')}`,
```

Comment:
I'm personally not a big fan because there should really be no circumstance in which they deviate. We already expect that you have to have the right `.env` files for deploy otherwise a lot of other things are going to break as well.

Reply:
Not super passionate, so if you aren't worried thats fine with me

I'm just imagining a situation where we want to upgrade attestation service because of a random bug but master has changes to the genesis generation since the last deploy & we then have to either checkout certain files to generate the correct genesis in the same way or to hardcode the genesis. Even though we definitely make assumptions in other places IMO the genesis should only be generated when the network is initially deployed

Reply:
But in that case you are deploying from `master` which you shouldn't be doing right? Because you might have other code, not just genesis generation code, that is incompatible?